iree/test/e2e/matmul/BUILD

```diff
@@ -193,3 +193,26 @@ py_binary(
 ) for lhs_rhs_type in [
     "f32",
 ]]
+
+[iree_generated_trace_runner_test(
+    name = "e2e_matmul_direct_f32_gpu_large_%s" % vulkan_target,
+    compiler_flags = [
+        "--iree-vulkan-target-triple=%s" % vulkan_target,
+    ],
+    generator = ":generate_e2e_matmul_tests",
+    generator_args = [
+        "--lhs_rhs_type=f32",
+        "--shapes=gpu_large",
+        "--compilation_info=SPIRVVectorize",
+    ],
+    tags = [
```

Comment:
Why is `requires-gpu-nvidia` here?   (Honest question - I never understood what this tag does already for the CUDA tests).  @ThomasRaoux @KoolJBlack ?

Reply:
It's for Google internal testing infrastructure purposes.